config.yml

```diff
@@ -6,6 +6,10 @@ accounts:
     mnemonic: "baby advance work soap slow exclude blur humble lucky rough teach wide chuckle captain rack laundry butter main very cannon donate armor dress follow"
     coins:
       - 999999999999999999upokt
+  - name: dao
```

Comment:
Good question. I #PUCed it up: `# PNF represents the multisig address that acts on behalf of the DAO`

We can be members of the DAO, but the foundation acts on our behalf.

When/if we move to true permissionless on-chain governance, this will go away.